mm/ktsan/thr.c

```diff
@@ -38,9 +39,13 @@ kt_thr_t *kt_thr_create(kt_thr_t *thr, int kid)
 		pr_err("KTSAN: maximum number of threads is reached\n");
 		BUG();
 	}
+	/* Kernel does not assign PIDs to some threads, give them fake
+	 * negative PIDs so that they are distinguishable in reports. */
+	if (pid == 0)
+		pid = --pool->new_pid;
```

Comment:
Are you sure?

The comment in rest_init says:
```
	/*
	 * We need to spawn init first so that it obtains pid 1, however
	 * the init task will end up wanting to create kthreads, which, if
	 * we schedule it before we create kthreadd, will OOPS.
	 */
```